samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatController.cs

```diff
@@ -78,10 +75,7 @@ public ChatController(ILogger<ChatController> logger)
         }
 
         // Register plugins that have been enabled
-        if (planner != null && plannerOptions.Value.Enabled)
-        {
-            await this.RegisterPlannerSkillsAsync(planner, plannerOptions.Value, openApiSkillsAuthHeaders, contextVariables);
-        }
+        await this.RegisterPlannerSkillsAsync(planner, openApiSkillsAuthHeaders, contextVariables);
```

Comment:
nit: Can you add an assert right before this call to fail when the planner is null